docs/sources/writing-guide/reuse-directories/index.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Reuse directories of content with Hugo mounts
+keywords:
+  - content reuse
+  - shared content
+  - Hugo
+menuTitle: Reuse directories of content
+title: Reuse directories of content with Hugo mounts
+---
+
+# Reuse directories of content with Hugo mounts
+
+> **Note:** The following task can only be performed by a Grafana Labs employee.
+
+Hugo mounts mount a source directory at a destination directory before the Hugo build.
+Use Hugo mounts to reuse a whole directory of content in another part of the website.
+
+## Before you begin
+
+- Identify a directory of content that should be reused.
+- Identify any pages to exclude.
+  Typically this is used to exclude pages that do not make sense in the context of the target directory.
+  > **Note:** Excluding files breaks any relative links to that page unless there is an equivalent in the target directory.
```

Comment:
```suggestion
  > **Warning:** Pages in the directory might have relative links to other pages that would be broken by their exclusion.
  > If the excluded page is replaced by another in the target directory, relative links will continue to work.
  > Otherwise, the broken relative links should be made absolute.
  > That way, they will always refer to the page in the source directory.
```